src/rgw/rgw_crypt.cc

```diff
@@ -1099,9 +1099,9 @@ int rgw_s3_prepare_encrypt(struct req_state* s,
       ldpp_dout(s, 5) << "Found RGW_ATTR_BUCKET_ENCRYPTION_POLICY on "
 	      << s->bucket_name << dendl;
 
-      bufferlist::const_iterator iter{&aiter->second};
+      RGWBucketEncryptionConfig bucket_encryption_conf;
       try {
-        RGWBucketEncryptionConfig bucket_encryption_conf;
+	bufferlist::const_iterator iter{&aiter->second};
```

Comment:
Spacing looks incorrect here.

Reply:
Will resolve all the spacing issues.

Reply:
Here I think the earlier spacing was wrong - now it is corrected.